notus/scanner/models/packages/deb.py

```diff
@@ -27,8 +27,8 @@
 
 from .package import Package, PackageComparison
 
-_deb_compile = re.compile(r"(.*)-(?:(\d*):)?(.*)-(.*)")
-_deb_compile_wo_revision = re.compile(r"(.*)-(?:(\d*):)?(.*)")
+_deb_compile = re.compile(r"(.*)-(?:(\d*):)?(\d.*)-(.*)")
+_deb_compile_wo_revision = re.compile(r"(.*)-(?:(\d*):)?(\d.*)")
```

Comment:
Could you write/adapt a unit test for that?

Reply:
done